app/views/layouts/application.html.erb

```diff
@@ -2,13 +2,88 @@
 <html>
 <head>
   <title>Betsy</title>
-  <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
-  <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
-  <%= csrf_meta_tags %>
+  <meta charset="utf-8">
+
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/css/bootstrap.min.css" integrity="sha384-1q8mTJOASx8j1Au+a5WDVnPi2lkFfwwEAa8hDDdjZlpLegxhjVME1fgjWPGmkzs7" crossorigin="anonymous">
+
+  <%=stylesheet_link_tag "application"%>
+
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.2/jquery.min.js"></script>
+  <script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.6/js/bootstrap.min.js"></script>
 </head>
 <body>
+  <main class="container">
+    <nav class="navbar navbar-default">
+      <div class="container-fluid">
+        <div class="navbar-header"><%=link_to "Betsy", root_path, class:"navbar-brand"%></div>
+        <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
+          <div class="col-lg-6">
+            <%= form_tag(search_product_path, method: "get", class: "navbar-form navbar-left") do %>
+            <div class="input-group">
+              <%= text_field_tag(:q, nil, class: 'form-control', placeholder: "Search a product") %>
+              <span class="input-group-btn">
+                <%= submit_tag("Search", class: "btn btn-default") %>
+                <% end %>
+              </span>
+            </div>
+          </div>
+          <ul class="nav navbar-nav navbar-right">
+            <li class="active"><%= link_to "All Products", products_path%></li>
+
+            <% if !current_user.nil? && current_user.id != nil %>
```

Comment:
@knaydee Did I step on anyone's toes by doing this Code Review. I've been out of the loop on reviewing and I'm sorry if you were planning on doing it and I took it from you. Seems to me that 80+ comments is enough for them to go through. So I'm concerned that additional comments from you would be overwhelming. Do projects usually have multiple reviewers? If you have any reasonable amount of background in javascript or CSS those are definitely weak points of the review that they might really appreciate your added feedback.